src/parser/parser.ts

```diff
@@ -119,3 +153,7 @@ export function renderDirectory(path: string): string {
         ),
     );
 }
+
+export function renderErrorPage(errorMessage: string): string {
+    return renderMarkdown(`# Something went wrong\n\n\`\`\`${errorMessage}\`\`\``);
+}
```

Comment:
Is the "Something went wrong" really a good header,

It could also be the standard:

- 404 Not Found
- 500 Internal Server Error

?

Reply:
I'd say it's good enough because we don't have proper status codes for a lot of cases so I wouldn't display them explicitly. But if you think that's better I'm also good with it so feel free to decide :)

Reply:
Yes that's good